packages/playwright-test/types/test.d.ts

```diff
@@ -475,7 +474,7 @@ interface TestConfig {
    *
    */
   webServer?: TestConfigWebServer;
-  plugins?: TestPlugin[],
+  plugins?: (TestPlugin | string | [string, any])[],
```

Comment:
All that nice typing…and then an `any` for the string load case?!

Reply:
I'll follow up with simplification!